.github/workflows/prerelease-tests-on-release-branch.yaml

```diff
@@ -0,0 +1,77 @@
+# Prerelease Tests
+
+name: Prerelease Tests on Release branch
+on:
+  workflow_dispatch:
+  schedule:
+    # run prerelease tests daily at 5 am UTC
+    - cron: '0 5 * * *'
+    # run coverity tests at 22:00 on every sunday
+    - cron: '0 22 * * SUN'
+
+# The workflow needs the permission to push branches
+permissions:
+  contents: write
+
+jobs:
+  prerelease_test:
+    name: Prerelease test
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout TimescaleDB
+        uses: actions/checkout@v4
+
+      - name: Set RELEASE_BRANCH as per environment
+        run: |
+          OLD_MINOR_VERSION=$(tail -1 version.config | cut -d ' ' -f 3 | cut -d '-' -f 1)
+          echo "RELEASE_BRANCH="$(echo ${OLD_MINOR_VERSION%?}x)"" >> $GITHUB_ENV
+
+      - name: Checkout TimescaleDB release branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.RELEASE_BRANCH }}
+
+# Prerelease Testing
+#
+# Push the release branch to upstream/prerelease_test branch in the TimescaleDB repository.
+#
+# Commits on this branch will be tested by more tests than our usual PRs.
+# You can see the pre-release test progress under the actions
+# (https://github.com/timescale/timescaledb/actions?query=branch%3Aprerelease_test)
+# or under the pull request you created for the release.
+#
+# Note: Because the upgrade script does not get executed at this stage, some test might fail.
+# These test failures can be ignored at this point. Your branch will be re-tested as soon as
+# the proper downgrade script is created and the versioning is adjusted.
+
+      - name: Push the release branch to prerelease_test branch
+        if: github.event.schedule == '0 5 * * *'
```

Comment:
This means that this action will not be executed for a `workflow_dispatch`. Actually, neither of the steps can do any work on a `workflow_dispatch`. Either you should not have `workflow_dispatch` (since it does not do anything), or you need to fix the condition so that it works for `workflow_dispatch` as well.

Reply:
Thanks, edited accordingly.